curious/__init__.py

```diff
@@ -68,15 +81,29 @@ def __add_model_by_class(self, cls, short_name=None):
       self.__short_names[manager.short_name].append(manager)
 
   def register(self, model, short_name=None):
-    if type(model) == types.ModuleType:
+    if isinstance(model, types.ModuleType):
       for name in dir(model):
         cls = getattr(model, name)
-        if inspect.isclass(cls) and issubclass(cls, django.db.models.Model) and cls._meta.abstract is False:
+        if (
+          inspect.isclass(cls)
+          and issubclass(cls, django.db.models.Model)
+          and not cls._meta.abstract
+        ):
           self.__add_model_by_class(cls)
     else:
-      if not hasattr(model, '_meta') or model._meta.abstract is False:
+      if not hasattr(model, '_meta') or not model._meta.abstract:
```

Comment:
This is not cosmetic and will fail for None as well.

Reply:
yes, this is intentional: `abstract = None` should _not_ behave as if it were abstract (not that anyone actually sets `abstract` to be that way). Again, since the use case for the difference in behavior is unlikely, and the result is idiomatic, I intend to keep this change.